code/modules/mob/living/carbon/human/species/shadekin/shadekin_abilities.dm

```diff
@@ -71,6 +71,9 @@
 	else if(stat)
 		to_chat(src, span_warning("Can't use that ability in your state!"))
 		return FALSE
+	else if(SK.doing_phase)
```

Comment:
Doesn't seem to be able to be triggered by normal means (spamming phase in/out button)